cluster/minio.go

```diff
@@ -124,6 +124,19 @@ func stopMinioTenantServers(sgt *StorageGroupTenant, tenantConf *TenantConfigura
 	return nil
 }
 
+func restartMinioTenantServers(sgt *StorageGroupTenant, tenantConf *TenantConfiguration) error {
+	adminClient, pErr := NewAdminClient(sgt.HTTPAddress(false), tenantConf.AccessKey, tenantConf.SecretKey)
+	if pErr != nil {
+		return pErr.Cause
+	}
+	// Restart minios after setting configuration
```

Comment:
comment should be just `restart` there is no config set.

Reply:
done